### PR TITLE
Extend scan for CustomResources to full index, instead of simply application index

### PR DESCRIPTION
**Pull request by metacosm**

### CODE REVIEW

File: extensions/kubernetes-client/deployment/src/main/java/io/quarkus/kubernetes/client/deployment/KubernetesClientProcessor.java [hunk 1]
```diff
@@ -10,11 +10,7 @@
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
-import org.jboss.jandex.AnnotationInstance;
-import org.jboss.jandex.AnnotationTarget;
-import org.jboss.jandex.ClassInfo;
-import org.jboss.jandex.DotName;
-import org.jboss.jandex.Type;
+import org.jboss.jandex.*;
```
Comment (gastaldi) [inline extensions/kubernetes-client/deployment/src/main/java/io/quarkus/kubernetes/client/deployment/KubernetesClientProcessor.java:13]: Avoid star imports

Comment (metacosm) [reply extensions/kubernetes-client/deployment/src/main/java/io/quarkus/kubernetes/client/deployment/KubernetesClientProcessor.java:13]: If I'm not mistaken, that's the work of the formatter, I didn't do that explicitly.

Comment (metacosm) [reply extensions/kubernetes-client/deployment/src/main/java/io/quarkus/kubernetes/client/deployment/KubernetesClientProcessor.java]: Indeed, my IDE was not properly configured… Thought that running `mvn -Dquickly` was formatting the code as well.

**Review by geoand — APPROVED** [review]

**Comment by metacosm** [comment]
Would be nice to backport this to 3.7 if possible, btw.

**Comment by geoand** [comment]
Is it really needed? We are trying to stabilize 3.7/3.8 as much as possible

**Comment by metacosm** [comment]
I guess I could add workaround for this in QOSDK until it's available in Quarkus.

**Comment by geoand** [comment]
If it's important and / or causes a lot work to workaround, we can have it. We are just trying to be as conservative as possible at the moment.

**Pull request merged**